### PR TITLE
review edit tests passing

### PR DESCRIPTION
**Pull request by harrisonmuskat**
wrote tests for allowing a user to edit their reviews

### CODE REVIEW

**Comment by harrisonmuskat** [comment]
[As an authenticated user, I want to edit my own reviews so that I can prevent others from seeing them](https://trello.com/c/D55ksmq6/32-as-an-authenticated-user-i-want-to-edit-my-own-reviews-so-that-i-can-prevent-others-from-seeing-them)

File: app/controllers/reviews_controller.rb [hunk 1]
```diff
@@ -25,4 +45,10 @@ def review_params
       user: User.find(current_user)
       )
   end
+
+  def authorize_user
+    if !user_signed_in?
+      raise ActionController::RoutingError.new("Not Found")
+    end
+  end
```
Comment (enthusiastick) [inline app/controllers/reviews_controller.rb]: You _may_ want to investigate the Devise helper method `authenticate_user` (although yours is doing a slightly different thing, which may be intentional design -- in which case, go you!)

File: spec/features/user_edits_review_spec.rb [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+  scenario "user cannot visit review edit page for someone else's review" do
+    user2 = FactoryGirl.create(:user)
+    login_as(user2, scope: :user)
+
+    visit show_path(@game_of_thrones)
+
+    expect(page).to_not have_link("edit")
+  end
+end
```
Comment (enthusiastick) [inline spec/features/user_edits_review_spec.rb]: This is a nice test.

File: spec/features/user_views_reviews_spec.rb [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      description: "watch this", rating: 2, show: rock_bottom, user: user
+    )
+
+    visit show_path(rock_bottom)
+
+    expect(page).to have_content rock_bottom.title
+    expect(page).to have_content review_for_rock.description
+
+    expect(page).not_to have_content game_of_thrones.title
+    # expect(page).not_to have_content review_for_thrones.rating
```
Comment (enthusiastick) [inline spec/features/user_views_reviews_spec.rb]: As a best practice, please try to avoid merging commented-out code into master.

**Pull request merged**